snakefile.py

```diff
@@ -39,13 +39,13 @@
 
 TRIMMED_READS_DIR = os.path.join(OUTPUT_DIR, 'trimmed_reads')
 LOG_DIR           = os.path.join(OUTPUT_DIR, 'logs')
-FASTQC_DIR        = os.path.join(OUTPUT_DIR, 'fastqc')
-MULTIQC_DIR       = os.path.join(OUTPUT_DIR, 'multiqc')
 MAPPED_READS_DIR  = os.path.join(OUTPUT_DIR, 'mapped_reads')
 VARIANTS_DIR      = os.path.join(OUTPUT_DIR, 'variants')
 KRAKEN_DIR        = os.path.join(OUTPUT_DIR, 'kraken')
 COVERAGE_DIR      = os.path.join(OUTPUT_DIR, 'coverage')
 REPORT_DIR        = os.path.join(OUTPUT_DIR, 'report')
+FASTQC_DIR        = os.path.join(REPORT_DIR, 'fastqc')
+MULTIQC_DIR       = os.path.join(REPORT_DIR, 'multiqc')
```

Comment:
I find this change curious.  I don't object to it, but it's curious to see these lines moved around.

Reply:
I needed it to be after `REPORT_DIR` is defined.

Reply:
d'oh!  Yeah, that's obvious.  Thanks!